Source/WebKit/UIProcess/API/Cocoa/WKWebViewInternal.h

```diff
@@ -247,7 +256,7 @@ struct PerWebProcessState {
 
     BOOL _findInteractionEnabled;
 #if HAVE(UIFINDINTERACTION)
-    RetainPtr<UIView> _findOverlay;
+    Vector<RetainPtr<UIView>> _findOverlaysForOverflowScroll; // Top, right, bottom, left
```

Comment:
The overlay ordering enforcement is a _little_ fragile... particularly since it's documented through a comment. What do you think about introducing a structure that captures what is going on?

```cpp
struct OverflowScrollOverlays {
    RetainPtr<UIView> top;
    RetainPtr<UIView> right;
    RetainPtr<UIView> bottom;
    RetainPtr<UIView> left;
};

OverflowScrollOverlays _findOverlaysForOverflowScroll;
```

Reply:
That way you can also get rid of `enum FindOverlayView`?